core/execution/future/market.go

```diff
@@ -2870,7 +3086,14 @@ func (m *Market) parkOrder(ctx context.Context, orderID string) *types.Order {
 			logging.Error(err))
 	}
 
-	_ = m.position.UnregisterOrder(ctx, order)
+	pos := m.position.UnregisterOrder(ctx, order)
+	if m.getMarginMode(order.Party) == types.MarginModeIsolatedMargin {
+		pos, _ := m.position.GetPositionByPartyID(pos.Party())
+		if err := m.updateIsolatedMarginOnOrder(ctx, pos, order); err != nil {
+			m.log.Panic("failed to update order margin after order park", logging.Order(order), logging.String("party", pos.Party()))
+		}
+	}
```

Comment:
Is that necessary? seeing that we do not support pegged order with isolated margin anymore, that should not be necessary, or be a panic as we should never end up in that position.

Reply:
it's not, I forgot to remove it when I disabled pegged